### ISSUE TITLE
Empty arrays not allowed in query

### ISSUE THREAD

**Issue by kliuless** [rmosolgo/graphql-ruby#69]
An empty array:
```
query test1 {
  testMe(array: [])
}
```
Causes this:
```
NoMethodError (undefined method `parts' for #<Hash:0x007ffa32b67f40>)
```

**Comment by rmosolgo** [comment]
Thanks for reporting this! I think it's fixed by f9bcce0, could you give it a try?

**Comment by kliuless** [comment]
Fix is good in f9bcce0.
